tests/example_apps/music/tables.py

```diff
@@ -24,14 +28,31 @@ def get_readable(cls) -> Readable:
         return Readable(template="%s", columns=[cls.name])
 
 
-class Band(Table):
-    name = Varchar(length=50)
-    manager = ForeignKey(Manager, null=True)
-    popularity = Integer(default=0)
+if engine.engine_type != "cockroach":  # type: ignore
 
-    @classmethod
-    def get_readable(cls) -> Readable:
-        return Readable(template="%s", columns=[cls.name])
+    class Band(Table):  # type: ignore
+        name = Varchar(length=50)
+        manager = ForeignKey(Manager, null=True)
+        popularity = Integer(default=0)
+
+        @classmethod
+        def get_readable(cls) -> Readable:
+            return Readable(template="%s", columns=[cls.name])
+
+else:
+
+    class Band(Table):  # type: ignore
+        """
+        Special version for Cockroach.
+        """
+
+        name = Varchar(length=50)
+        manager = ForeignKey(Manager, null=True)
+        popularity = BigInt(default=0)
+
+        @classmethod
+        def get_readable(cls) -> Readable:
+            return Readable(template="%s", columns=[cls.name])
```

Comment:
What happens if someone uses `Integer` with Cockroach? Should we make `Integer` just maps to `BigInt` automatically, by adding `column_type` to `BigInt`?

Reply:
Same story explained above for `JSON` vs `JSONB`

The developer can use `Integer` or `BigInt`, it does not matter, but Cockroach will always return BigInt, so the test here fails, and we need the special table.

Reply:
Added docs: https://github.com/piccolo-orm/piccolo/pull/608/commits/ebe06e7e0286e4c2484bbd626740435eb50297f9